packages/sys-apps/dstat/files/Dool-fix-for-some-Python-deprecations.patch

```diff
@@ -0,0 +1,48 @@
+Source: https://github.com/scottchiefbaker/dool/ (dstat fork)
+
+From 0caaccc1b2faf70e23bee937b528eab14880dd14 Mon Sep 17 00:00:00 2001
+From: Scott Baker <scott@perturb.org>
+Date: Sun, 14 Jul 2019 18:30:10 -0700
+Subject: [PATCH] Dool fix for some Python deprecations
+
+---
+ dstat | 9 +++++++--
+ 1 file changed, 7 insertions(+), 2 deletions(-)
+
+diff --git a/dstat b/dstat
+index 9ad7db6..bad8c1b 100755
+--- a/dstat
++++ b/dstat
+@@ -19,7 +19,6 @@
+ from __future__ import absolute_import, division, generators, print_function
+ __metaclass__ = type
+
+-import collections
+ import fnmatch
+ import getopt
+ import getpass
+@@ -33,6 +32,12 @@ import six
+ import sys
+ import time
+
++# https://stackoverflow.com/questions/53978542/how-to-use-collections-abc-from-both-python-3-8-and-python-2-7
++try:
++    from collections.abc import Sequence # Python 3.8+
```

Comment:
DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working